mosdepth.nim

```diff
@@ -125,7 +151,7 @@ iterator gen_quantized(quants: seq[int], arr: coverage_t): depth_s {.inline.} =
         yield (last_pos, pos, lookup[last_quantized])
       last_quantized = quantized
       last_pos = pos
-    if last_quantized != -1 and last_pos < arr.high and last_quantized < len(lookup):
+    if last_quantized != -1 and last_pos < arr.high:
```

Comment:
make sure you dont' change this. It was a recent bug-fix.

Reply:
Thanks I put those fixes back. Did not realize you had updated things.